shared/actions/tracker2.js

```diff
@@ -193,7 +193,7 @@ const showUser = (_, action) => {
     reason: '',
   })
 
-  if (flags.useNewRouter) {
+  if (flags.useNewRouter && !action.payload.skipNav) {
```

Comment:
profile on remount would call reload which does this which can try and push a screen. that should never nav so add a param